eks/modules/services/cert-manager.tf

```diff
@@ -1,22 +1,40 @@
+resource "local_file" "le_cert_issuer" {
+  content = templatefile("${path.module}/le-cert-issuer.yaml.tmpl", {
+    email = var.email,
+    hosted_zone_id = var.hosted_zone_id,
+    region = var.region,
+    zone = var.hosted_zone_name
+  })
+  filename = "${path.module}/le-cert-issuer.yaml"
+}
+
 resource "kubernetes_namespace" "cert_manager" {
+  depends_on = [kubernetes_config_map.aws_auth]
+
   metadata {
     name = "cert-manager"
 
     labels = {
-      "certmanager.k8s.io/disable-validation" = "true"
+      "cert-manager.io/disable-validation" = "true"
     }
   }
-  depends_on = [kubernetes_config_map.aws_auth]
+
+  timeouts {
+    delete = "30m"
+  }
```

Comment:
It doesn't hurt but probably not needed for the cert-manager NS. The reason we are a bit conservative with long timeouts for the CAP namespaces are there are so many objects in these namespaces it takes time for k8s to clean up but the same doesn't likely hold true for cert-manager.